docs/use-cases/Nvidia-GPU-passthrough-and-Kata.md

```diff
@@ -67,7 +67,7 @@ To use large BARs devices (for example, Nvidia Tesla P100), you need Kata versio
 
 The following configuration in the Kata `configuration.toml` file as shown below can work:
 
-Hotplug for PCI devices by `shpchp` (Linux's SHPC PCI Hotplug driver):
+Hotplug for PCI devices by `acpi_pcihp` (Linux's ACPI PCI Hotplug driver):
```

Comment:
I'd feel more comfortable if we had an ack from Nvidia (or atleast someone with that hardware who can verify this) about the changes to this file.

Reply:
Fwiw, note that the initial comment there was already incorrect.  Although we used SHPC on the qemu side, the guest kernel did *not* use SHPC due to a weird quirk of the configuration we used.  And, if you enabled SHPC on the guest side, everything broke.  Instead we were doing hotplug by "shove it in, then force a rescan and hope nothing breaks".